pkg/api/types.go

```diff
@@ -926,6 +914,12 @@ const (
 	// Status code 422
 	StatusReasonInvalid StatusReason = "Invalid"
 
+	// StatusReasonTimeout means that the request could not be completed within the given time.
```

Comment:
Constants should always be duplicated in the event we have to covert them - someone who depends on constant from specific version should be able to rely on that in code.


> On Jan 27, 2015, at 2:21 PM, Nikhil Jindal <notifications@github.com> wrote:
> 
> In pkg/api/types.go:
> 
> > @@ -926,6 +914,12 @@ const (
> >  	// Status code 422
> >  	StatusReasonInvalid StatusReason = "Invalid"
> >  
> > +	// StatusReasonTimeout means that the request could not be completed within the given time.
> Yes. It wasnt clear to me why some of the constants are duplicated in versioned types.
> It didnt look like they are needed there and hence I didnt add them.
> 
> —
> Reply to this email directly or view it on GitHub.
>